content/docs/join_us/recruitment_FAQs.md

```diff
@@ -241,6 +241,24 @@ This list is not exhaustive, but some examples include:
 - Research programming support (running [drop-in sessions]({{< relref "docs/regular_events/drop-in_sessions" >}}) for people at the Turing)
 - Knowledge sharing (organising our [tech talks]({{< relref "docs/regular_events/lunchtime_tech_talks.md" >}}))
 
+### Can you tell me more about what it is like working at The Alan Turing Institute?
+
+At The Turing, we want to make sure you feel comfortable and supported in how you work. Here’s a bit about what to expect:
+
+- **Hybrid Working**: We only ask that you come into the office four days a month — the rest is up to you! Some of the team work primarily remotely, while others come in every day. There is no pressure or expectation to be in the office more often than the minimum four days.
```

Comment:
I fully agree with *"There is no pressure or expectation"* and I'd add that everyone in REG is incredibly accommodating of remote working (this really impressed me when I started here). However, while remote working is *fully* supported, the Turing definitely also has a culture of "discussing topics around the watercooler" which can lead to FOMO if you're not in the office relatively frequently. This at least has been my experience.

Reply:
Also very true. I'll add something, thanks David!

Reply:
We could also highlight that we do try and mitigate this a little with things like randomized coffee chats and co-working sessions? I also really liked the E&S away day in that hybrid was really prioritised, and the organisers tried to make a first-class experience for remote participants.

I do generally agree with David's comment, though.

Reply:
We can livestream the lunch area during lunch hours to mitigate this!